### ISSUE TITLE
Request: Create packages for distribution

### ISSUE THREAD

**Issue by Cryowatt** [quintilesims/layer0#128]
Why doesn't layer0 distribute itself as a package instead of a zip?

Windows - Chocolatey
Mac - Homebrew
Linux - RPM and/or DEB

**Comment by diemonster** [comment]
Good idea, no reason other than we haven't done it yet. Would love to get your help on the Chocolatey distribution. 

http://i3.cpcache.com/product/696364158/lets_get_chocolatey_bumper_stickers.jpg

**Comment by Cryowatt** [comment]
Ideally if you move to a packaged-based distribution method, you should add terraform as a package dependency instead of redistributing their binaries as part of the L0 release.

**Comment by diemonster** [comment]
@Cryowatt pinging you for template that you've worked on already for this?

**Comment by nehayward** [comment]
https://askubuntu.com/questions/16446/how-to-get-my-software-into-ubuntu

We are going to have to use https://launchpad.net to distribute on linux. I think this means a separate code base to maintain
